packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -261,7 +261,9 @@ abstract contract StandardBridge is Initializable {
         onlyOtherBridge
     {
         require(paused() == false, "StandardBridge: paused");
-        require(isCustomGasToken() == false, "StandardBridge: cannot bridge ETH with custom gas token");
+        if (isCustomGasToken()) {
+            require(msg.value == 0, "StandardBridge: cannot bridge ETH with custom gas token");
+        }
```

Comment:
There isn't a good reason to allow for these APIs at all in custom gas token mode. They do nothing when `msg.value == 0` so should revert to the previous assertion